draft-high-assurance-profile-oid4vc-sd-jwt-vc.md

```diff
@@ -176,40 +185,36 @@ All the requirements defined in [@!I-D.ietf-oauth-selective-disclosure-jwt] MUST
+Note: this requirement might change when claim-based and biometrics-based holder binding become available.
 
-Note: In some credential types, it is not desirable to include an expiration date (eg: diploma attestation). Therefore, this profile leaves its inclusion to the Issuer, or the body defining the respective credential type.
+Note: Re-using the same Credential across Verifiers, or re-using the same JWK value across multiple Credentials gives colluding Verifiers a mechanism to correlate the User. There are currently two known ways to address this with SD-JWT VCs. First is to issue multiple instances of the same credentials with different JWK values, so that if each instance of the credential is used at only one Verifier, it can be reused multiple times. Another is to use each credential only once (ephemeral credentials). It is RECOMMENDED to adopt one of these mechanisms.
 
-## Issuer identification and key resolution to validate an issued Credential {#issuer-key-resolution}
+Note: If there is a requirement to communicate information about the verification status and identity assurance data of the claims about the subject, the syntax defined by @!OIDF.ekyc-ida SHOULD be used. It is up to each jurisdiction and ecosystem, whether to require it to the implementers of this profile.
 
-* When `iss` claim is an HTTPS URL, the key used to validate the Issuer’s signature on the SD-JWT VC MUST be hosted at the .well-known path as defined in draft-terbu-sd-jwt-vc. JOSE header `kid` MUST be used to obtain the key.
-* When the `iss’ identifier is a Subject Alternative Name (SAN), and the key used to validate the Issuer’s signature on the SD-JWT VC MUST be obtained from the `x5c` JOSE header.
+Note: If there is a requirement to provide the Subject’s identifier assigned and maintained by the Issuer, `sub` claim MAY be used. There is no requirement for a binding to exist between `sub` and `cnf` claims. See section X in @!I-D.draft-terbu-sd-jwt-vc for implementation considerations.
 
-Note: The issuer MAY decide to support both options. In which case, it is at the discretion of the Wallet and the Verifier which key to use for the issuer signature validation.
+Note: In some credential types, it is not desirable to include an expiration date (eg: diploma attestation). Therefore, this profile leaves its inclusion to the Issuer, or the body defining the respective credential type.
 
-## Holder key resolution {#holder-key-resolution}
+## Issuer identification and key resolution to validate an issued Credential {#issuer-key-resolution}
 
-* The public key passed in the `cnf` claim [@!RFC7800] MUST be a JSON Web Key [@!RFC7517] contained in the `jwk` claim. `kid' MUST be included in the JWK in the `cnf` claim, since it will be used in the JOSE Header.
+This profile supports two ways to represent and resolves the key required to validate the issuer signature of a SD-JWT-VC, web-based key resolution and x.509 certificates.
```

Comment:
```suggestion
This profile supports two ways to represent and resolves the key required to validate the issuer signature of a VC-SD-JWT, web PKI-based key resolution and x.509 certificates.
```

Reply:
Why would you call it web PKI? I think web based key resolution is sufficient.

Reply:
it's trust in the DNS sytem, that allows to trust the .well-known path. i thought it should be clearer